src/components/itineraries/Generate/Generate.tsx

```diff
@@ -137,17 +123,14 @@ const Generate: FC = () => {
                 }
               </Div>
               <Div>
-                <p>Hoeveel adressen wil je in je looplijst? (Max. 20)</p>
-                <div>
-                  <FormField
-                    name="num"
-                    component={StyledInput}
-                    type="number"
-                    min='1'
-                    max='20'
-                    step='1'
-                  />
-                </div>
+                <div><Label label='Hoeveel adressen wil je in je looplijst? (Max. 20)' /></div>
+                <StyledNumberControl
+                  min={1}
+                  max={20}
+                  step={1}
+                  name='num'
+                  validate={combineValidators(isRequired, isBetween(1, 20))}
```

Comment:
Could be nice to read the `isBetween` arguments from the `min` and `max` props. Is that possbile?

Reply:
Done